app/components/Admin/Admin.js

```diff
@@ -39,63 +39,66 @@ class Admin extends React.Component {
     dataService.get('/api/change_requests', getAuthRequestHeaders()).then((json) => {
       this.setState({
         change_requests: json.change_requests,
-        changeRequestsLoaded: true
+        changeRequestsLoaded: true,
       });
     });
   }
 
+  bulkActionHandler(action, changeRequests) {
+    return Promise.all(
+      changeRequests.map((changeRequest) => { // Create an action handler for each CR
+        const changeRequestFields = changeRequest.field_changes.reduce((a, c) => { // reduce fields into one object
+          a[c.field_name] = c.field_value;
```

Comment:
Good idea. Done, will log a warning if there is a duplicate, we should probably eventually support some popup/push notification architecture so non devs can debug issues like that can be a followup issue.